assignments/prototype-refactor.js

```diff
@@ -7,3 +7,139 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+/*
+  === GameObject ===
+  * createdAt
+  * name
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+
+class GameObject {
+  constructor(object) {
```

Comment:
Well done at refactoring your code with classes. Your code is really well written.
The only thing that I can suggest is to change the parameter name in something more meaningful (e.g `player` or `character`). You could even use something like `playerObj` if you want to make it clear an object needs to be passed there.